db/migrate/20210328233843_fix_bookmarks_with_incorrect_topic_id.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
```

Comment:
what do you think about moving that to post_migrate?

Reply:
Oh right I always forget that things like this should go there, cause stuff can still mess up while deploying. Will move it